datasets/ch/seco_sanctions/crawler.py

```diff
@@ -6,6 +6,7 @@
 from followthemoney.types import registry
 from followthemoney.util import join_text
 from lxml.etree import _Element as Element
+from slugify import slugify
```

Comment:
Can you use `from normality import slugify`? It's all over the rest of our codebase.